rfcs/text/0000-multi-data-collection-dataset-processing.md

```diff
@@ -0,0 +1,233 @@
+*   A search query accompanied by a timestamp (to guarantee reproducibility and immutability)
+*   Other future representations as project needs evolve (e.g. linking to a particular project bundle version)
+
+Critically, the submitted DAPS will not explicitly dictate the files that have to be co-processed in pipeline instances as this would require specialized knowledge of the pipelines by the submitter. Instead, **a DAPS indicates that aggregation of data must be performed only from the list of data provided (submission envelope), but the actual aggregation is delegated to the process of analysis initiation, subject to well defined modality-specific metadata requirements**. 
+
+The imposition of specific requirements to metadata is part of this RFC proposal and a key component of the proposed DAPS solution.
+
+This approach provides a generalized signaling mechanism suitable for all foreseeable data types while accounting for data type-specific requirements.
+
+The DAPS approach has common elements with project-level bundles that have been proposed in other contexts, but differs in the following ways:
+
+1. It does not impose any specific aggregation level on the data allowing for more flexibility and accommodating future unforeseen processing scenarios
+2. Allows specification of bundles included in a DAPS in many alternative different ways (listing, search, etc…)
+3. Does not impose the creation of a new bundle type that is tied to a specific data modality
+4. Dictates specific metadata requirements that guarantee the correct processing of all foreseeable data modalities
+5. Decouples analysis initiation from data representation allowing for future flexibility
+
+## General Metadata Requirements
+In the above framework, analysis pipelines will receive a set of bundles that _potentially_ need to be co-processed. As aforementioned, this signal will not explicitly identify the bundles that need to be co-processed in a single pipeline, instead, it will serve as a notification **indicating that any co-processing is to be done only within the boundaries of this set of files (submission envelope). **
+
+Analysis can initiate multiple pipeline invocations as a result of a single DAPS being submitted.
+
+The actual grouping of the files is to be performed by analysis via a search in the metadata of the bundles submitted following well-defined modality-specific set of rules. For this reason, the metadata must satisfy the following metadata requirements (MR):
+
+Furthermore, if part of the data submitted in a DAPS has already been processed the Analysis infrastructure will only re-process the parts that have not been modified.
+
+1. The data modality of each data file included in the DAPS must be clearly identifiable (MR1)
+2. Data that need to be processed together must be clearly identifiable by a modality-specific set of metadata variables that will group the files according to processing requirements. (MR2)
```

Comment:
Please give an example of specific metadata fields which could be used to trigger an analysis?

Reply:
examples are in a later section of the document

Reply:
It would be great to see them here, having to flip back and forth in a document to understand its meaning makes it very difficult to process.